audio/stereo_panner.rs

```diff
@@ -0,0 +1,139 @@
+use block::{Block, Chunk, Tick, FRAMES_PER_BLOCK};
+use euclid::Vector3D;
+use node::{AudioNodeEngine, AudioNodeMessage, BlockInfo};
+use node::{AudioNodeType, ChannelInfo};
+use param::{Param, ParamDir, ParamType};
+use std::f32::consts::PI;
+use std::cmp::min;
+use std::cmp::max;
+
+// .normalize(), but it takes into account zero vectors
+use panner_node::normalize_zero;
+
+#[derive(Copy, Clone, Debug)]
+pub struct StereoPannerOptions {
+    pub pan: f32,
+}
+
+pub enum StereoPannerNodeMessage {
+    SetPan(f32),
+}
+
+impl Default for StereoPannerOptions {
+    fn default() -> Self {
+        StereoPannerOptions {
+            pan: 0.,
+        }
+    }
+}
+
+#[derive(AudioNodeCommon)]
+pub(crate) struct StereoPannerNode {
+    pan: Param,
+}
+
+impl StereoPannerNode {
+    pub fn new(options: StereoPannerOptions, channel_info: ChannelInfo) -> Self {
```

Comment:
You'll need to update `AudioNodeInit` to have `StereoPannerOptions` inside its `StereoPannerNode` variant